src/components/Button/Button.tsx

```diff
@@ -4,13 +4,14 @@ import styles from './Button.module.css';
 import { Icon } from '../Icon/Icon';
 export interface Props {
   /**
-   * Button reference
+   * Visually hidden button text (but text is still accessible to assistive technology).
+   * This overrides `text`
    */
-  buttonRef?: any;
+  'aria-label'?: string;
   /**
-   * Toggle between full width button (true) and default button width
+   * Button reference
    */
-  block?: boolean;
+  buttonRef?: any;
```

Comment:
nit: are we getting rid of this in a future PR? think we'd prefer to have the component `forwardRef` to people can pass in `ref` directly